WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -285,14 +300,16 @@ extension DashboardPromptsCardCell: BlogDashboardCardConfigurable {
     func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
         self.blog = blog
         self.presenterViewController = viewController
-        refreshStackView()
+        fetchPrompt()
```

Comment:
Probably not a big deal and just something to note - I noticed this gets called every time the device goes from portrait to landscape. I didn't investigate too much but it looks like the dashboard might be getting recreated and that eventually ends up calling this on a view load.

Reply:
> I noticed this gets called every time the device goes from portrait to landscape. I didn't investigate too much but it looks like the dashboard might be getting recreated and that eventually ends up calling this on a view load.

Thanks for noting this. It's not typical behavior so it didn't occur to me. 

> Probably not a big deal

If we were only going to get prompts via fetching, this is something we'd definitely not want to do. But once prompts are cached, it won't be so offensive. In the end, we should fetch only if we don't have a cached prompt. So the fetch should happen once per prompt. In theory. 😄